File: src/main/java/com/team06/hanq/entity/CultureInfo.java
package com.team06.hanq.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "culture_information")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CultureInfo {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long infoId;

    @Column(nullable = false)
    private String title;

    private String subtitle;
    private String subsubtitle;

    @Column(nullable = false, columnDefinition = "TEXT")
    private String explanation;
}

File: lib/DTO/culture_info.dart
class CultureInfo {
  final int infoId;
  final String title;
  final String subtitle;
  final String subsubtitle;
  final String explanation;

  CultureInfo({
    required this.infoId,
    required this.title,
    required this.subtitle,
    required this.subsubtitle,
    required this.explanation,
  });

  factory CultureInfo.fromJson(Map<String, dynamic> json) {
    return CultureInfo(
      infoId: (json['infoId'] as num).toInt(),
      title: json['title'] ?? '',
      subtitle: json['subtitle'] ?? '',
      subsubtitle: json['subsubtitle'] ?? '',
      explanation: json['explanation'] ?? '',
    );
  }
}
